lib/newrelic-couchbase/instrumentation/bucket.rb

```diff
@@ -1,34 +1,29 @@
 ::Couchbase::Bucket.class_eval do
-  include NewRelic::Agent::MethodTracer
-
-  # https://code.google.com/p/memcached/wiki/BinaryProtocolRevamped#Introduction
   [
     :get,
     :set,
+    :cas,
+    :compare_and_swap,
     :add,
     :replace,
     :delete,
+    :incr,
     :increment,
+    :decr,
     :decrement,
     :flush,
     :append,
     :prepend,
     :touch,
     :stats,
-    :run
+    :run,
+    :design_docs,
+    :save_design_doc,
+    :delete_design_doc,
+    :observe_and_wait,
+    :create_timer,
+    :create_periodic_timer
   ].each do |instruction|
-    add_method_tracer instruction, "Couchbase/Bucket/#{instruction.to_s}"
+    NewRelic::Agent::Datastores.trace self, instruction, "Couchbase"
```

Comment:
In the newrelic-riak gem, we're actually tracking the name of the bucket as well.  Do we get that for free or do we need to use the `NewRelic::Agent::Datastores.wrap` api?

Reply:
Actually looking at your newrelic link, it looks like we don't get that for free.  In Riak, you can see we say "Riak D find" where "D" is the bucket name.